jablib/src/test/java/org/jabref/logic/util/ExternalLinkCreatorTest.java

```diff
@@ -37,6 +37,39 @@ void getShortScienceSearchURLEncodesSpecialCharacters() {
         assertTrue(urlIsValid(url.get()));
     }
 
+    @Test
+    void getShortScienceSearchURLEncodesUnicodeCharacters() {
+        BibEntry entry = new BibEntry().withField(StandardField.TITLE, "歷史書 📖 📚");
+        Optional<String> url = getShortScienceSearchURL(entry);
+        assertTrue(url.isPresent());
+        assertTrue(urlIsValid(url.get()));
+
+        // Expected behaviour is to link to the search results page, /internalsearch, and the unicode and emojis are percent-encoded
+        assertEquals(Optional.of("https://www.shortscience.org/internalsearch?q=%E6%AD%B7%E5%8F%B2%E6%9B%B8%20%F0%9F%93%96%20%F0%9F%93%9A"), url);
+    }
+
+    @Test
+    void getShortScienceSearchURLTrimsWhitespaceInTitle() {
+        BibEntry entry = new BibEntry().withField(StandardField.TITLE, "    History Textbook   ");
+        Optional<String> url = getShortScienceSearchURL(entry);
+        assertTrue(url.isPresent());
```

Comment:
Remove, because you check for equalaty below - and the test below also covers optional precense